libs/vkd3d/utils.c

```diff
@@ -647,6 +648,34 @@ VkFormat vkd3d_get_vk_format(DXGI_FORMAT format)
     return VK_FORMAT_UNDEFINED;
 }
 
+/* get some size-based low bits for prioritization in the same way as d3d12 */
+inline uint32_t vkd3d_get_priority_adjust(VkDeviceSize size)
+{
+    return min((size / (10 * 1048576)), 0xFFFFUL);
```

Comment:
This is magical. Is the 10 MiB bin-size arbitrary, or is this actually documented?